scripts/docker/install-docker.ps1

```diff
@@ -1,102 +1,18 @@
 # https://msdn.microsoft.com/de-de/virtualization/windowscontainers/quick_start/quick_start_windows_10
 Set-ExecutionPolicy Bypass -scope Process
-New-Item -Type Directory -Path 'C:\Program Files\docker\'
-Invoke-WebRequest https://aka.ms/tp5/b/dockerd -OutFile $env:ProgramFiles\docker\dockerd.exe
-Invoke-WebRequest https://aka.ms/tp5/b/docker -OutFile $env:ProgramFiles\docker\docker.exe
-[Environment]::SetEnvironmentVariable("Path", $env:Path + ";C:\Program Files\Docker", [EnvironmentVariableTarget]::Machine)
+Invoke-WebRequest "https://master.dockerproject.org/windows/amd64/docker-1.13.0-dev.zip" -OutFile "$env:TEMP\docker-1.13.0-dev.zip" -UseBasicParsing
+Expand-Archive -Path "$env:TEMP\docker-1.13.0-dev.zip" -DestinationPath $env:ProgramFiles
```

Comment:
@jhowardmsft - do you recommend v1.12.1 on TP5 instead? I thought the fixes in v1.13 also apply to TP5/14300